### ISSUE TITLE
Inquirer.js now accepts input/output options

### ISSUE THREAD

**Issue by ruyadorno** [knownasilya/cli-width#2]
Hi @knownasilya, while I was playing with Inquirer's new input/output streams options (https://github.com/SBoudrias/Inquirer.js/pull/280) I stumbled upon some problems with **cli-width** making direct access to the `process.stdout` stream - As you can see in the file changes from the linked PR, the output stream now can be user defined and not only `process.stdout`. 

Now, I don't want to force any design/api but it would be really nice if the module could accept an object as the function parameter, that let us set the `output` stream (it can always default to `process.stdout`) and even defining the `defaultWidth` value too - I find it quite non-standard the way we define the default value as a property before invoking the function.

Let me know if a PR for these changes is welcome and most importantly, many thanks for all the good work :trophy:

**Comment by ruyadorno** [comment]
relates to https://github.com/SBoudrias/Inquirer.js/issues/285

**Comment by knownasilya** [comment]
Sounds like a nice change. Would totally accept a PR. Like you said, it should default to what it is now, even if options aren't specified.

I'm not a fan of the way that the default width is currently set, so I do like your idea of setting it in the options hash (removing the way that the default width is currently set), but that would probably mean that we have to push a major version, and update inquirer to use that version as well.

``` js
cliWidth({
  output: customOut,
  defaultWidth: 200,
  tty: require('customtty') // custom tty, not sure if that would be useful.
});
```

**Comment by knownasilya** [comment]
@ruyadorno are you still planning to submit a PR for this change?

**Comment by ruyadorno** [comment]
yes :blush: I'm quite busy with personal matters for now so feel free to implement it if you have the time - if you don't, I'll be happy to submit the PR once everything settles for me :+1:

**Comment by knownasilya** [comment]
Released as v2.0.0
